src/format.js

```diff
@@ -0,0 +1,3 @@
+export function toUSD(number) {
```

Comment:
awesome job with this function!! It's great that you're identifying repeated code and breaking it out into functions - it shows that you're critically thinking about what you're doing.